FurnitureShop/src/FurnitureShop.Client.Api/Services/OrderService.cs

```diff
@@ -0,0 +1,53 @@
+﻿using FurnitureShop.Client.Api.Dtos;
+using FurnitureShop.Client.Api.Services.Interfaces;
+using FurnitureShop.Client.Api.ViewModel;
+using FurnitureShop.Common.Exceptions;
+using FurnitureShop.Data.Context;
+using FurnitureShop.Data.Entities;
+using FurnitureShop.Data.Repositories;
+using Mapster;
+using Microsoft.EntityFrameworkCore;
+
+namespace FurnitureShop.Client.Api.Services;
+
+public class OrderService : IOrderService
+{
+    private readonly AppDbContext _context;
+    private readonly UnitOfWork _unitOfWork;
+
+    public OrderService(UnitOfWork unitOfWork, AppDbContext context)
+    {
+        _unitOfWork = unitOfWork;
+        _context = context;
+    }
+
+    public async Task<OrderView> CreateOrder(CreateOrderDto createOrderDto)
+    {
+        if (createOrderDto is null)
+            throw new BadRequestException("createOrderDto is not null");
```

Comment:
create order dto is null 
not kerak emas

Reply:
yoq kere

Reply:
teshirish kerekiuuuuuu???

Reply:
OSHA gapdagi not sozi kerak emas

Reply:
haaa